### PR TITLE
Editing the fund your training section

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/OT7pOR3Y/4144-revise-page-titles-for-organic-search-funding-and-support-pages

### Context

We need to revise the page titles in this section of the website for SEO (and for user experience).

We also agreed in our navigation workshop to add a subcategory to the fund your support page.

Talks with the UCD team and Laura (international CD) have also led me to remove the 'if you come from outside England' card as it sets false expectations for users about what funding they can expect to receive. I've added this info to the tuition fees and maintenance loans page instead.

### CODE REVIEW

File: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md [hunk 1]
```diff
@@ -39,6 +39,30 @@ You will only have to make loan repayments once you’re earning.
...
+* [funding in Guernsey](https://www.gov.gg/article/152744/Policies)
+* [funding in the Isle of Man](https://www.gov.im/student-grants)
+
+### Non-UK citizens
+
+If you’re not a UK citizen, you are unlikely to get help funding your training unless you have permission to [live permanently in the UK](https://www.gov.uk/browse/visas-immigration/settle-in-the-uk), for example under the EU Settlement Scheme, or are training to teach languages or physics. You will also need to meet some other eligibility requirements.
+
+[Find out about funding for non-UK citizens](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
+
 ## Check your eligibility for a scholarship or bursary
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md:66]: this heading feels a bit chunky @EllieNodder (particularly give you have to go elsewhere for the info) - worth shortening?

**Review by gemmadallmandfe — COMMENTED** [review]
This looks great @EllieNodder - makes much more sense now

I've left one comment which isn't actually on your changes so happy for you to ignore and get this into prod today

File: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md (hunk 1 above)
Comment (EllieNodder) [reply app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md:66]: worth getting these changes live now and thinking about tweaking this after?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**